src/providerhub/HISTORY.rst

```diff
@@ -3,6 +3,24 @@
 Release History
 ===============
 
+0.2.0
+++++++
+* Support commands for
+`az providerhub notification-registration`
+`az providerhub operation`
+`az providerhub sku`
+
+* Enable creation of nested resource types and adding extension categories.
+
+Refine and update initial release commands.
+* [BREAKING CHANGE] `az providerhub manifest checkin`:
+`--arm-manifest-location` renamed to `--baseline-arm-manifest-location` or `--manifest-location`
```

Comment:
I think this kind of breaking change is unnecessary. Reserving the argument alias `--arm-manifest-location` is simple by codegen configuration. https://github.com/Azure/autorest.az#add-alias-for-a-parameter